mesh/mesh.cpp

```diff
@@ -2534,7 +2534,11 @@ Mesh::Mesh(const Mesh &mesh, bool copy_nodes)
    }
 
    // Copy the vertices
-   mesh.vertices.Copy(vertices);
+   vertices.SetSize(NumOfVertices);
+   for (int i = 0; i < NumOfVertices; i++)
+   {
+      vertices[i].SetCoords(spaceDim, mesh.vertices[i]());
+   }
```

Comment:
Why this change?

Reply:
Fix a GCC 8.1 warning, see fa6fdb7

Reply:
Oh, I think I fixed this warning by making class `Vertex` trivially copyable. It used to have an empty, explicitly defined destructor which was making it non-trivially copyable as defined by the c++ standard ([TriviallyCopyable](https://en.cppreference.com/w/cpp/named_req/TriviallyCopyable)). The fix is in 265006d59f which in now merged in master.

Reply:
I have dropped my commit

Reply:
Thanks Stefano